### DISCUSSION TITLE
Can I use the QT client to control a remote Transmission daemon?

### DISCUSSION THREAD

**Discussion by itcultus** [transmission/transmission#4522] [Q&A] [answered]
As the title says :)
I have a transmission daemon running on a server. And I use the web UI to manage it. 
My plan is to build a new container-based headless daemon and then use the QT-client to manage it. 
Is it possible?

**Comment by ckerr** [comment, marked as answer]
Yes, transmission-qt can remote control transmission-daemon. I use that feature pretty much every day. :smile_cat:

**Reply by zhen-huan-hu** [reply]
I think we really need to do a better advertisement of this feature. Very few people realize that the QT client actually can function as a thin client like Deluge. A few years ago, when I built my headless server, I had two requirements for the client: 1) remote access 2) able to rename files and directories. I ended up choosing Deluge because Transmission Web UI (even to this day) does not support file renaming and I assumed the QT client cannot function as a thin client based on my experience with the Mac client.

**Reply by ckerr** [reply]
What do you suggest?

**Reply by zhen-huan-hu** [reply]
> What do you suggest?

On the front page of transmissionbt.com, instead of just saying "The Qt-based Windows UI has been modernized and is fully Windows 11 ready.", maybe we could also point out it can function as a thin client? Like "The QT-based Windows UI has been designed to run as both a standalone Windows application and a thin-client to connect to a Transmission daemon running on a remote server. Now, it has been modernized and is fully Windows 11 ready".

As a reference, this is on Deluge website: "Deluge has been designed to run as both a normal standalone desktop application and as a ​client-server. In Thinclient mode a Deluge daemon handles all the BitTorrent activity and is able to run on headless machines with the user-interfaces connecting remotely from any other platform."

**Reply by ckerr** [reply]
@zhen-huan-hu sounds like a good idea. Do you want to make a PR to https://github.com/transmission/transmission.github.io to write it up?

**Reply by zhen-huan-hu** [reply]
I just created a PR: https://github.com/transmission/transmission.github.io/pull/76

**Reply by o-alquimista** [reply]
What about the GTK client? I couldn't find this feature in it.

**Reply by melyux** [reply]
Or the Mac client?

**Reply by wally-crunk** [reply]
The Mac client does not have this feature yet.
